src/settings/index.ts

```diff
@@ -14,10 +14,31 @@ import { VenueTemplate } from "types/venues";
 
 import { FIVE_MINUTES_MS } from "utils/time";
 
+import asteroid from "assets/backgrounds/Asteroid.jpg";
+import bigIsland from "assets/backgrounds/BigIsland.jpg";
+import corridor from "assets/backgrounds/Corridor.jpg";
+import cubes from "assets/backgrounds/Cubes.jpg";
+import earth from "assets/backgrounds/Earth.jpg";
+import field from "assets/backgrounds/Field.jpg";
+import fireSide from "assets/backgrounds/FireSide.jpg";
+import floatyCity from "assets/backgrounds/FloatyCity.jpg";
+import halloween from "assets/backgrounds/Halloween.png";
 import defaultMapIcon from "assets/icons/default-map-icon.png";
 import sparkleNavLogo from "assets/icons/sparkle-nav-logo.png";
 import sparkleverseLogo from "assets/images/sparkleverse-logo.png";
 
+export const DEFAULT_BACKGROUNDS = [
+  asteroid,
+  bigIsland,
+  corridor,
+  cubes,
+  earth,
+  field,
+  fireSide,
+  floatyCity,
+  halloween,
+];
+
```

Comment:
Can move added lines from here to `src/settings/map-backgrounds.ts` and add re-export like  https://github.com/sparkletown/sparkle/blob/0d7caca456e5d867976fcac061a01a7edd132c48/src/settings/index.ts#L21


This will ease off clutter like in the previously single `settings.ts` file.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2327/commits/89b37f939f50e6e776b17ca3c10e9198e890c897